src/components/Admin/Admin.js

```diff
@@ -43,19 +37,19 @@ class Admin extends Component {
       crossDomain: true,
       success: function(response) {
         // console.log(response.showAdmin);
-        if (response.showAdmin !== true) {
+        if (response.showAdmin) {
           this.setState({
-            showNotAdminDialog: true,
+            isAdmin: true,
```

Comment:
Looks good. But I would suggest to make a small code optimization. Please use ternary operator inside setState. Extra 4-5 lines of code can be removed :)